api.js

```diff
@@ -0,0 +1,164 @@
+"use strict";
+
+const {classes: Cc, interfaces: Ci, results: Cr, utils: Cu} = Components;
+
+Cu.import("resource://gre/modules/XPCOMUtils.jsm");
+
+Cu.import("resource://devtools/shared/event-emitter.js");
+Cu.import("resource://gre/modules/ExtensionUtils.jsm");
+const {SingletonEventManager} = ExtensionUtils;
+
+XPCOMUtils.defineLazyModuleGetter(this, "webrtcUI",
+                                  "resource:///modules/webrtcUI.jsm");
+
+let origReceiveMessage = null;
```

Comment:
I've only touched that module lightly. I think @fqueze is a better person to answer that.